packages/upload-api/src/access/confirm.js

```diff
@@ -111,6 +111,11 @@ export async function createSessionProofs({
     capabilities,
     expiration,
     proofs: delegationProofs,
+    facts: [
+      {
+        service: service.did(),
+      }
+    ]
```

Comment:
I realized after we spoke that it would be a better idea to include a link to the invocation that caused it specifically `access/confirm` capability we received here, which is created at 

https://github.com/web3-storage/w3up/blob/b232aa784c178cc6877982d7d9dfe4da06a06203/packages/upload-api/src/access/authorize.js#L51-L71

In other words I think it will be something like


```suggestion
    facts: [{ cause: invocation.cid }]
```

It also would make sense to add a link to `access/authorize` to a `access/confirm` we create, but that could also be a separate pr.

Reply:
I added a fact like the one you suggest